gatsby-node.js

```diff
@@ -90,6 +90,16 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
         }
       }
 
+      whatsNewPosts: allMarkdownRemark(
+        filter: { fileAbsolutePath: { regex: "/src/content/whats-new/" } }
```

Comment:
I'm also gathering this has some redirect work! would you be able to explain what is going on here? or maybe a separate PR for the redirect stuff?

Reply:
Nevermind! I just read the full PR! Sorry about that

Reply:
Ah right, I kinda snuck in a separate ticket in this PR. This fixes #670 to make sure the old whats new URLs map to the new ones. I did this work in this PR because this PR [was failing](https://github.com/newrelic/docs-website/actions/runs/522497102) due to some weird cache issues from deleting all of the data dictionary stuff. By changing `gatsby-node`, this force-cleared the cache and allowed this PR to build successfully. Sorry about the confusion here!